Source/Diagnostics/FlushFormats/FlushFormatPlotfile.cpp

```diff
@@ -302,7 +302,10 @@ FlushFormatPlotfile::WriteWarpXHeader(
 
         warpx.GetPartContainer().WriteHeader(HeaderFile);
 
-        HeaderFile << warpx.getcurrent_injection_position() << "\n";
+        for (auto& inj_pos : warpx.getcurrent_injection_position())
```

Comment:
```suggestion
        for (auto const& inj_pos : warpx.getcurrent_injection_position())
```
Tools like clang-tidy might not like it, even though it's not a big deal in this particular case.

Reply:
Actually you may have to use `auto const&` or `auto` but not `auto &`, because the function returns `Real` by value.

Reply:
Oh, I was looking at the old WarpX code. The new code return `Vector<Real> const&`. Anyway, I think it makes sense to use `auto const&`.

Reply:
Got it, done in [72b2ee2](https://github.com/ECP-WarpX/WarpX/pull/3958/commits/72b2ee27f99ff6489044c8fe84476693da2aff81).